management/templates/system-backup.html

```diff
@@ -111,6 +112,31 @@ <h3>Configuration</h3>
       <input type="text" class="form-control" rows="1" id="backup-target-pass">
     </div>
   </div>
+  <!-- Backblaze -->
+  <div class="form-group backup-target-b2">
+      <div class="col-sm-10 col-sm-offset-2">
+      <p>Backups are stored in a Backblaze B2 bucket. You must have a Backblaze account already.</p>
```

Comment:
Maybe a link to backblaze would help?
```suggestion
      <p>Backups are stored in a <a href="https://www.backblaze.com/" target="_blank" rel="noreferrer">Backblaze</a> B2 bucket. You must have a Backblaze account already.</p>
```

Reply:
Yes 👍